examples/http-certification/json-api/src/backend/backend.did

```diff
@@ -8,13 +8,69 @@ type HttpRequest = record {
     certificate_version : opt nat16;
 };
 
+type HttpUpdateRequest = record {
+    method : text;
+    url : text;
+    headers : vec HeaderField;
+    body : blob;
+};
+
 type HttpResponse = record {
     status_code : nat16;
     headers : vec HeaderField;
     body : blob;
+    upgrade : opt bool;
```

Comment:
if we have a custom type above for the request that drops the `certificate_version` for update calls, i would also have another type for the response that drops the `upgrade`, otherwise i would keep in the request as well.

Reply:
Good point. I opted to just drop it for now because I realized that we don't have the support on the Rust side for those types, so I added a ticket to handle that: https://dfinity.atlassian.net/browse/TT-392